core/variable_model.js

```diff
@@ -73,3 +76,13 @@ Blockly.VariableModel = function(name, opt_type, opt_id) {
 Blockly.VariableModel.prototype.getId = function() {
   return this.id_;
 };
+
+/**
+ * A custom compare function for the VariableModel objects.
+ * @param {Blockly.VariableModel} var1 First variable to compare.
+ * @param {Blockly.VariableModel} var2 Second variable to compare.
```

Comment:
Add to description: 
@return {number} -1 if name of var1 is less than name of var2, 0 if equal,
     and 1 if greater.